alphagov/paas-cf
# [#175872849] Offer new, even smaller, Redis 5.x plans

What
----
Smaller Redis 5.x plans (the smallest possible) were requested by Department for Education so they could reduce their spend. The new plans use `cache.t3.micro` instances on AWS, offering 0.5gb memory.

Billing configuration changes are included here.

How to review
-------------
1. Deploy to your env
2. Check you can provision the new plans

---

🚨⚠️ Please do not merge this pull request via the GitHub UI ⚠️🚨
